view/.eslintrc.yml

```diff
@@ -31,6 +31,7 @@ rules:
     # Stylistic Issues
     require-jsdoc: 0
     spaced-comment: 0
+    max-statements: 20
```

Comment:
in eslint first param is reporting level (0 unreported, 1 warn, 2 error)
second param is config.
so this should read

``` yml
max-statements:
    - 2
    - 20
```